code/src/languages/python/web/web_console.py

```diff
@@ -115,7 +119,7 @@ def format(self):
 def print_tb():
     trace = Trace()
     traceback.print_exc(file=trace)
-    sys.stderr.write(trace.format())
+    err(trace.format())
```

Comment:
Where does the `err` come from?

Reply:
Line 151, same file

Reply:
It's a wrapper around `stderr.write` (as opposed to `sys.stderr.write`). Basically, using `sys.stderr` doesn't work because python does weird stuff with it, but `stderr` connects directly to the process stderr.

Reply:
Ah thanks, I see it now, I thought I'd expanded the files in Github when grepping for it but clearly not. Thanks for clarifying!